### PR TITLE
Updated the jar filename in target/

### PR DESCRIPTION
**Pull request by emcfins**
*Issue #, if available:*
Was getting an error when running ./run-java-demoapp.sh

`Error: Could not find or load main class com.amazonaws.kinesisvideo.demoapp.DemoAppMain
Caused by: java.lang.ClassNotFoundException: com.amazonaws.kinesisvideo.demoapp.DemoAppMain`

*Description of changes:*

Updated the jar filename in the run-java-demoapp.sh file to reflect the files created from the compile command.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,6 +21,8 @@ The Amazon Kinesis Video Streams Producer SDK Java makes it easy to build an on-
 
 * You can find available pre-built KinesisVideoProducerJNI library in [src/main/resources/lib/](https://github.com/awslabs/amazon-kinesis-video-streams-producer-sdk-java/tree/master/src/main/resources/lib) for Mac (x64), Ubuntu (x64) and Raspian (x86) and Windows 10. If pre-built libraries did not work for you, ["KinesisVideoProducerJNI"](https://github.com/awslabs/amazon-kinesis-video-streams-producer-sdk-cpp) native library needs to be built first before running the Java demo application. Please follow the steps  in the section **Build the native library (KinesisVideoProducerJNI) to run Java Demo App** in Producer SDK CPP [readme](https://github.com/awslabs/amazon-kinesis-video-streams-producer-sdk-cpp).
 
+* You need to have a Kinesis Video Stream created in your AWS account.
```
Comment (zhiyua-git) [inline README.md]: This is not needed, the stream will be created if given stream name is not existing.

File: README.md [hunk 2]
```diff
@@ -76,15 +78,15 @@ Refer the **README.md** file in the  *dockerscripts* folder for running the buil
 
 #### Launching PutMediaDemo sample application
 
- Run `PutMediaDemo.java` to send sample mkv stream to Kinesis Video Streams. **Note:** ACCESS_KEY and SECRET_KEY are required for running this sample application as well. However, this demo application does not require JNI.
+ Run `PutMediaDemo.java` to send sample mkv stream to Kinesis Video Streams. **Note:** ACCESS_KEY, SECRET_KEY, and an existing KINESIS_VIDEO_STREAM are required for running this sample application as well. However, this demo application does not require JNI.
```
Comment (zhiyua-git) [inline README.md]: Same as above.

File: README.md [hunk 3]
```diff
@@ -31,12 +33,12 @@ Import the Maven project to your IDE, it will find dependency packages from Mave
 
 Run `DemoAppMain.java` in `./src/main/demo` with JVM arguments set to
 ```
--Daws.accessKeyId=<YourAwsAccessKey> -Daws.secretKey=<YourAwsSecretKey> -Djava.library.path=<NativeLibraryPath>
+-Daws.accessKeyId=<YourAwsAccessKey> -Daws.secretKey=<YourAwsSecretKey> -Dkvs-stream=<YourKinesisVideoStream> -Djava.library.path=<NativeLibraryPath>
```
Comment (zhiyua-git) [inline README.md]: YourKinesisVideoStreamName

**Comment by jelling** [comment]
If the point of this library is to get developers feet wet so they will use Kinesis, having broken setup instructions is not going to help. Please at least merge the fix to the instructions to save the time of troubleshooting this.

**Pull request closed without merging**